src/main/java/com/pikachu/purple/bootstrap/perfume/controller/PerfumeController.java

```diff
@@ -18,20 +19,20 @@ public class PerfumeController implements PerfumeApi {
     private final PerfumeGetByUserPreferenceNoteUseCase perfumeGetByUserPreferenceNoteUseCase;
 
     @Override
-    public GetPerfumeByBrandsResponse getPerfumeByBrands(GetPerfumeByBrandsRequest request) {
-        PerfumeGetByBrandsUseCase.Result result = perfumeGetByBrandsUseCase.invoke(new Command(request.brandList()));
+    public SuccessResponse<GetPerfumeByBrandsResponse> getPerfumeByBrands(List<String> request) {
+        PerfumeGetByBrandsUseCase.Result result = perfumeGetByBrandsUseCase.invoke(new Command(request));
 
-        return new GetPerfumeByBrandsResponse(result.perfumeList());
+        return SuccessResponse.of(new GetPerfumeByBrandsResponse(result.perfumeList())) ;
     }
 
     @Override
-    public GetPreferenceBasedRecommendResponse getPreferenceBasedRecommend() {
+    public SuccessResponse<GetPreferenceBasedRecommendResponse> getPreferenceBasedRecommend() {
```

Comment:
p3) 해당 함수명을 봤을 때, 정확히 무엇을 가져오는지 파악이 조금 어려운 것 같아요. getAllPerfumeByPreference() 요런 뉘앙스는 어떨까요?
```suggestion
    public SuccessResponse<GetPreferenceBasedRecommendResponse> getPreferenceBasedRecommend() {
```